ledger/participant-integration-api/src/main/scala/platform/store/backend/common/EventStorageBackendTemplate.scala

```diff
@@ -86,6 +87,7 @@ abstract class EventStorageBackendTemplate(
       "NULL as create_key_hash",
       "create_key_value_compression",
       "submitters",
+      "NULL as driver_metadata",
```

Comment:
Flat transactions have 'drive_metadata' always empty?

Reply:
this is the exercise event parsing where we need to artificially fill in this column so it can be parsed as a generic resultset (e.g. we have the same treatment for all columns pertaining to create events , e.g. see previous lines)

Reply:
@pbatko-da - it looks like the columns for `baseColumnsForFlatTransactionsCreate` and `baseColumnsForFlatTransactionsExercise` need to match as they use the same row parser.